cyences_app_for_splunk/default/macros.conf

```diff
@@ -282,6 +282,12 @@ definition = 10
 definition = search NOT index IN ("cyences", "audit_summary", "endpoint_summary", "gia_summary", "notable", "cim_modactions", "risk")
 iseval = 0
 
+[cs_disk_usage(1)]
+args = host
+definition = index="_introspection" sourcetype=splunk_disk_objects component=Partitions  host IN ($host$) | eval server = host | eval mount_point = 'data.mount_point' | eval free = if(isnotnull('data.available'), 'data.available', 'data.free') | eval pct_disk_usage = (1 - free / 'data.capacity') * 100 | eval high_priority = if(free <=`cs_minimum_disk_available_thresold` OR pct_disk_usage>85,1,0 ) | bin _time span=10min | eval serverp=server."----".mount_point
```

Comment:
Let's create macro for disk percentage as well.

[cs_maximum_disk_percentage_thresold]
definition = 85